demisto_sdk/commands/common/tests/docker_test.py

```diff
@@ -169,6 +169,7 @@ def test_is_docker_image_latest_tag_with_default_image(self):
         docker_image_validator.is_modified_file = False
         docker_image_validator.docker_image_tag = '1.3-alpine'
         docker_image_validator.is_valid = True
+        docker_image_validator.yml_docker_image = True
```

Comment:
This is not a possible value according to the code, it is better to mock a possible value.
Fix in all places please.